system/Config/Services.php

```diff
@@ -363,21 +344,16 @@ public static function honeypot(BaseConfig $config = null, bool $getShared = tru
 	 *
 	 * @return \CodeIgniter\Images\Handlers\BaseHandler
 	 */
-	public static function image(string $handler = null, $config = null, bool $getShared = true)
+	public static function image(string $handler = null, Images $config = null, bool $getShared = true)
 	{
 		if ($getShared)
 		{
 			return static::getSharedInstance('image', $handler, $config);
 		}
 
-		if (empty($config))
-		{
-			$config = new Images();
-		}
-
-		$handler = is_null($handler) ? $config->defaultHandler : $handler;
-
-		$class = $config->handlers[$handler];
+		$config  = $config ?? config('Images');
+		$handler = $handler ?: $config->defaultHandler;
+		$class   = $config->handlers[$handler];
 
 		return new $class($config); // @phpstan-ignore-line
```

Comment:
`// @phpstan-ignore-line` no longer needed.

```suggestion
		return new $class($config);
```

Reply:
Thanks.